src/github_utils.py

```diff
@@ -19,111 +20,107 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
```

Comment:
The original docstring was too vague. Updated the docstring for clarity and included type hints.
```suggestion
|
```
N/A